### PR TITLE
Feature/create token steps

### LINKED ISSUE #939: create token steps design

### PR DESCRIPTION
**Pull request by AtanasKrondev**
Closes #939 

Design adjustments on the create token step by step page.
Raw text moved to wording.

### CODE REVIEW

**Comment by thepiwo** [comment]
I think we should stick in some parts to the old color scheme until it is updated in develop, otherwise the difference will look so hard, also, e.g. background colors will be set by global color scheme.

**Comment by thepiwo** [comment]
long names break layout 
image

**Comment by thepiwo** [comment]
margin doesn't look right

File: src/components/CreateToken.vue [hunk 1]
```diff
@@ -1,75 +1,136 @@
...
-          minlength="1"
-          maxlength="500"
-          :disabled="loadingState"
+          v-for="i in 6"
+          :key="i"
+          class="step"
         >
+          <div
+            class="step-box"
+            :class="{ active: step >= i - 1, pulse: ((step === i - 1 && step !== 0) || step > 6) }"
```
Comment (thepiwo) [inline src/components/CreateToken.vue:34]: this looks quite complicated, what was the issue with the previous implementation?

**Comment by AtanasKrondev** [comment]
> long names break layout

Could you tell on what system and browser

**Review by thepiwo — APPROVED** [review]
great work, thanks!

**Pull request merged**